browser/ui/webui/brave_wallet/page_handler/wallet_page_handler.cc

```diff
@@ -31,3 +31,7 @@ void WalletPageHandler::ShowApprovePanelUI() {
 void WalletPageHandler::ShowWalletBackupUI() {
   NOTREACHED_IN_MIGRATION();
 }
+
+void WalletPageHandler::UnlockWalletUI() {
+  NOTREACHED_IN_MIGRATION();
```

Comment:
New code should use `NOTREACHED()`. 
It's ok to replace NOTREACHED_IN_MIGRATION()->NOTREACHED() for consistency.